ui/appui-react/src/appui-react/statusbar/popup/StatusBarPopover.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module StatusBar
+ */
+
+import * as React from "react";
+import { Popover } from "@itwin/itwinui-react";
+import { ButtonExpandIndicator } from "./ExpandIndicator";
+
+/** Popover component used in `StatusBar` component.
+ * This component should wrap the element that triggers the popover.
+ * @note Add the `StatusBarPopover.ExpandIndicator` to popover trigger buttons.
+ * @public
+ */
+export function StatusBarPopover(props: React.ComponentProps<typeof Popover>) {
```

Comment:
I'd prefer to keep the props type "smaller", we can add additional properties if needed.

Reply:
I added a props interface with the required props.